content/audio.md

```diff
@@ -17,50 +21,98 @@ section: software-troubleshooting
 
-| PulseAudio | PipeWire |
-| :--------- | :-------|
-| Ubuntu pre-22.10 and Pop!\_OS pre-22.04 | Ubuntu 22.10+ and Pop!\_OS 22.04+ |
-| `systemctl --user restart pulseaudio` | `systemctl --user restart wireplumber pipewire pipewire-pulse` |
-| `rm -r ~/.config/pulse` |  `rm -r ~/.config/pulse` |
-| `pulseaudio -k` | |
+Go to <u>Launcher</u> and search for Sound:
 
-This set of commands first restarts the sound daemon and removes the user's configuration for PulseAudio. On systems still using PulseAudio as a server, it restarts the PulseAudio server, which will create new default audio configuration files.
+![audio_1](images/audio/audio_1.png)
+
+Under the Sound Settings, the Input and Output Device selected will be displayed:
+
+![audio_2](images/audio/audio_2.png)
+
+## Verify Audio Devices Detected by the System
+
+Below are set of commands that can be used to detect various information about your audio device.
+
+List all detected audio devices:
+
+```
+aplay -l
+```
+
+List audio input devices:
+
+```
+arecord -l
+```
+
+List the current sound cards:
+
+```
+cat /proc/asound/cards
+```
+
+List kernel modules related to audio:
+
+```
+sudo lspci -k | grep -A 3 -i audio
+```
+
+Your audio devices should appear in these lists.
+
+If not, this may be a driver issue.
+
+## Restart the Audio Services
+
+Restart the audio daemon:
+
+```
+systemctl --user restart wireplumber pipewire pipewire-pulse
+```
+```
+rm -r ~/.config/pulse
+```
+
+This set of commands restarts the <u>PipeWire audio server</u> and its associated services for the current user. This also removes the <u>PulseAudio</u> configuration and will create new default audio configuration files.
 
 ## Check the PulseAudio Controls
 
-The program <u>PulseAudio Volume Control</u> is helpful in figuring out which program is producing audio, where that audio is being routed, what the default input/output devices are, and what the volume levels are set to.  It can be installed using the Pop!\_Shop, or with this command:
+The program <u>PulseAudio Volume Control</u> is helpful in figuring out which program is producing audio, where that audio is being routed, what the default input/output devices are, and what the volume levels are set to. It can be installed using the Pop!_Shop or Cosmic Store:
+
+![audio_3](images/audio/audio_3.png)
+
+or with this command:
 
 ```
 sudo apt install pavucontrol
 ```
 
-Once installed, open the <u>PulseAudio Volume Control</u> app in the Activities or Applications menu, or run `pavucontrol` in a terminal.
+Once installed, open the PulseAudio Volume Control app in the Activities or Applications menu, or run pavucontrol in a terminal.
 
 The "Playback" tab will show a list of applications that are currently playing audio. The volume for each application can be individually controlled.
 
-![Playback tab](/images/audio/pavucontrol-playback.png)
+![audio_4](images/audio/audio_4.png)
```

Comment:
Please update both the filenames and the captions for each of these images to be more useful.

Filenames should be grouped by stem (e.g. `pavucontrol-playback.png` and `pavucontrol-recording.png`). There's no need to use underscores in these file names, they're just harder to type than hyphens.

Captions should be e.g. "Playback tab in PulseAudio Volume Control" or "Playback tab in pavucontrol". This is what will show up if the image fails to load, or if a blind user reads the page with a screen reader.